src/app/resize/resize.component.ts

```diff
@@ -1,50 +1,140 @@
-import {Component, ElementRef, HostListener} from '@angular/core';
+/*
+Resize component - used to resize passed components horizontally or vertically.
+Horizontal resize is the default.
+Inputs:
+direction - options: 'vertical', 'horizontal'. Default 'horizontal'
+collapsed - options: true / false. Default false. Used for components that have expanded and collapsed state
+disabled - options true / false. Default false. Useful when displaying sets of elements where only some of the have resize property.
```

Comment:
I'd inline input comments and keep the description. 
Also have a star at the beginning of every line would be nice: 
```
/*
 * bla
 * bla
 */